benches/tpcc/schema.lua

```diff
@@ -0,0 +1,144 @@
+local sql_execute = box.sql and box.sql.execute or box.execute
```

Comment:
I strongly against of duplicating of a schema for a benchmark across several repositories. Is there any other solution?

Reply:
The solution would be to patch tpcc repository, and I think that's what we should do in the future. This was copied to make tpcc be able to start from prepared snapshot (as loading the data takes so much time). I will remove this code and add a big ticket in backlog for cleaning up the mess about the code that splits in different repositories

Reply:
Removed